packages/block-editor/src/components/block-list/block.js

```diff
@@ -187,7 +188,7 @@ function BlockListBlock( {
 		name,
 		mode,
 		blockTitle: blockType.title,
-		wrapperProps,
+		wrapperProps: omit( wrapperProps, [ 'data-align' ] ),
```

Comment:
Not for this PR but that whole object "value" is mostly important for the wrapper hook and not for the current component.

Reply:
Yes, that's true. The memoized value as context should ideally be reduced or even avoided entirely.